Software build (tag 2.2) is broken when NEOPIXEL_ENABLE is not set

The software build is broken with the latest tag (2.2) when disabling the neopixel feature.

The build fails with an error message like this:
```
src/Led.cpp: In function 'void Led_SetNightmode(bool)':
src/Led.cpp:185:6: error: 'Led_NightMode' was not declared in this scope
  if (Led_NightMode == enabled) {
      ^~~~~~~~~~~~~
```

The reason for this is that in src\Led.cpp the methods "Led_SetNightmode", "Led_GetNightmode", "Led_ToggleNightmode" do not contain a pair of "#ifdef NEOPIXEL_ENABLE" and "#endif" as the other methods do.

--- a/src/Led.cpp
+++ b/src/Led.cpp
@@ -182,6 +182,7 @@ void Led_SetBrightness(uint8_t value) {
 }
 
 void Led_SetNightmode(bool enabled) {
+#ifdef NEOPIXEL_ENABLE
 	if (Led_NightMode == enabled) {
 		// we don't need to do anything
 		return;
@@ -198,14 +199,19 @@ void Led_SetNightmode(bool enabled) {
 	Led_NightMode = enabled;
 	Led_SetBrightness(newValue);
 	Log_Println(msg, LOGLEVEL_INFO);
+#endif
 }
 
 bool Led_GetNightmode() {
+#ifdef NEOPIXEL_ENABLE
 	return Led_NightMode;
+#endif
 }
 
 void Led_ToggleNightmode() {
+#ifdef NEOPIXEL_ENABLE
 	Led_SetNightmode(!Led_NightMode);
+#endif
 }
 
 // Calculates physical address for a virtual LED address. This handles reversing the rotation direction of the ring and shifting the starting LED

